setup.py

```diff
@@ -51,7 +51,7 @@ def get_extras_require() -> Dict[str, List[str]]:
     requirements = {
         # TODO(HideakiImamura) Unpin mypy version after fixing "Duplicate modules" error in
         # tutorials.
-        "checking": ["black", "hacking", "isort", "mypy==0.790", "blackdoc"],
+        "checking": ["black", "hacking", "isort", "mypy==0.790", "blackdoc", "toml"],
```

Comment:
The update in this PR might be removed in the near future, so could you add a TODO line like

```suggestion
        # TODO(xadrianzetx ) Remove `toml` when https://github.com/keewis/blackdoc/issues/100 is resolved
        "checking": ["black", "hacking", "isort", "mypy==0.790", "blackdoc", "toml"],
```
?